docs/src/pages/layout/portal.md

```diff
@@ -8,7 +8,7 @@ The portal component renders its children into a new "subtree"
 outside of current component hierarchy.
 The children of the portal component will be appended to the `container` specified.
 
-The component is used interally by our [`Modal`](/api/modal) component.
+The component is used internally by our [`Modal`](/api/modal) component.
```

Comment:
Could you also change "our" to "the"? Thanks!

Reply:
Sure